scripts/sonar/sonar-build.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -e
+
+# Configure the make
+aclocal && autoconf && autoheader && automake --add-missing --copy
+./configure --quiet --with-rnd --with-services --with-uitests
+
+# Make with the sonar build watcher
+build-wrapper-linux-x86-64 --out-dir bw-output make -sj$(nproc)
```

Comment:
If `hoot-services` were to be scanned, you'd need the database variables configured (cause `liquibase` gets called deep in there somewhere on build).